src/builder.rs

```diff
@@ -657,10 +664,22 @@ pub mod testing {
             ),
             rng_seed in prop::array::uniform32(prop::num::u8::ANY)
         ) -> ArbitraryBundleInputs<StdRng> {
+            let mut tree = BridgeTree::<MerkleHashOrchard, 4>::new(100);
```

Comment:
Copy-paste error; this will generate a small tree with paths that are the wrong length for the circuit.
```suggestion
            let mut tree = BridgeTree::<MerkleHashOrchard, MERKLE_DEPTH_ORCHARD>::new(100);
```